### PR TITLE
ENGDOCS-1994

### PR DESCRIPTION
**Pull request by aevesdocker**
Add more admin settings for 4.28 

https://www.notion.so/dockerinc/Admin-Settings-manage-file-sharing-options-8ec4083f8e6442febfc1ef2bdf1f6079
https://www.notion.so/dockerinc/Admin-settings-file-sharing-type-5ceb0c1fe5c14686a7c1e476b71c3a9e

## Reviews

- [x] Technical review 
- [x] Editorial review
- [x] Product review

### CODE REVIEW

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 1]
```diff
@@ -166,6 +189,9 @@ The following `admin-settings.json` code and table provides an example of the re
 | `allowExperimentalFeatures`| | If `value` is set to `false`, experimental features are disabled.|
 | `allowBetaFeatures`| | If `value` is set to `false`, beta features are disabled.|
 | `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](../../../engine/reference/commandline/image_load.md) and receive an error if they try to.|
+| `filesharingAllowedDirectories` |  | Specify which paths your developers can add file shares to. Accepts `$HOME`, `$TMP`, or `$TEMP` as `path` variables. This specifices the allowed parent directory and subdirectories. If `sharedByDefault` is set to `true`, only the resolved paths should will be on the allow list. |
```
Comment (neonima) [inline content/desktop/hardened-desktop/settings-management/configure.md:192]: Specify which paths your developers can add file shares to. Also accepts env $HOME, $TMP, or $TEMP as path variables. When a path is added, its subdirectories are allowed. If sharedByDefault is set to true, that path will be added upon factory reset or when DD first starts on the machine

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 2]
```diff
@@ -166,6 +189,9 @@ The following `admin-settings.json` code and table provides an example of the re
...
 | `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](../../../engine/reference/commandline/image_load.md) and receive an error if they try to.|
+| `filesharingAllowedDirectories` |  | Specify which paths your developers can add file shares to. Also accepts `$HOME`, `$TMP`, or `$TEMP` as `path` variables. When a path is added, its subdirectories are allowed. If `sharedByDefault` is set to `true`, that path will be added upon factory reset or when Docker Desktop first starts. |
+| `useVirtualizationFrameworkVirtioFS`|  <span class="badge badge-info">macOS only</span> | If `value` is set to `true`, VirtioFS is set as the file sharing mechanism. Note: If both `useVirtualizationFrameworkVirtioFS` and `useGrpcfuse` have `value` set to `true`, VirtioFS takes precedence. Likewise, if both `useVirtualizationFrameworkVirtioFS` and `useGrpcfuse` have `value` set to `false`, osxfs is set as the file sharing mechanism. |
+| `useGrpcfuse` | <span class="badge badge-info">macOS only</span> | If `value` is set to `true`, gRPC Fuse is set as the file sharing mechanism. |
```
Comment (dvdksn) [inline content/desktop/hardened-desktop/settings-management/configure.md:194]: the `span` elements don't do anything in this table so I suggest we remove them, or if you want to have badges you can use the `badge` shortcode

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 3]
```diff
@@ -138,22 +138,45 @@ The following `admin-settings.json` code and table provides an example of the re
   "blockDockerLoad": {
     "locked": false,
     "value": true
+  },
+  {
```
Comment (neonima) [inline content/desktop/hardened-desktop/settings-management/configure.md:142]: Remove this and its closing one

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 4]
```diff
@@ -138,22 +138,45 @@ The following `admin-settings.json` code and table provides an example of the re
...
+	"configurationFileVersion": 2,
```
Comment (neonima) [inline content/desktop/hardened-desktop/settings-management/configure.md:143]: Remove, it's already at the top :)

File: content/desktop/hardened-desktop/settings-management/configure.md [hunk 5]
```diff
@@ -138,22 +138,40 @@ The following `admin-settings.json` code and table provides an example of the re
...
+	"filesharingAllowedDirectories": [
+	  {
+	    "path": "$HOME",
+	    "sharedByDefault": true
+	  },
+	  {
+	    "path":"$TMP",
+	    "sharedByDefault": false
+	  }
+	],
```
Comment (dvdksn) [inline content/desktop/hardened-desktop/settings-management/configure.md:151]: nit: looks like there's a mix of tabs and spaces in the indentation here, which could lead to strange behavior when rendered. I suggest we use spaces
```suggestion
  "filesharingAllowedDirectories": [
    {
      "path": "$HOME",
      "sharedByDefault": true
    },
    {
      "path":"$TMP",
      "sharedByDefault": false
    }
  ], 
```

**Review by dvdksn — APPROVED** [review]
LGTM

**Review by craig-osterhout — APPROVED** [review]

**Pull request merged**